database/migrations/2014_10_12_000001_modify_users_table.php

```diff
@@ -20,6 +20,12 @@ public function up()
             $table->text('bio')->nullable()->after('password'); // Will be written in Markdown. The user profile image will come from Gravatar account for the email address.
             $table->text('title')->nullable()->after('bio');
             $table->softDeletes()->after('title');
+
+            // needed for Laravel Socialite
+            $table->string('password')->nullable()->change();
```

Comment:
I don't want a password to be nullable. We have it automatically create a password for user registration, so that would be fine for Socialite user creation.